tests/unit_tests/gui/conftest.py

```diff
@@ -52,12 +52,12 @@
 
 def with_manage_tool(gui, qtbot: QtBot, callback) -> None:
     def handle_manage_dialog():
-        dialog = wait_for_child(gui, qtbot, ClosableDialog, name="manage-cases")
+        dialog = wait_for_child(gui, qtbot, ClosableDialog, name="manage-experiments")
         cases_panel = get_child(dialog, CaseInitializationConfigurationPanel)
         callback(dialog, cases_panel)
 
     QTimer.singleShot(1000, handle_manage_dialog)
-    manage_tool = gui.tools["Manage cases"]
+    manage_tool = gui.tools["Manage experiments"]
```

Comment:
"Manage storage"?

Reply:
Not sure we want users to think about "Storage" at all.
What do you think @oyvindeide  and @frode-aarstad ?

Reply:
Not sure whats best. Is there a better word that can cover both experiments and ensembles? Eventually, we probably want the user to be able to delete experiments and ensembles in this tool aswell.

But "Manage experiments" is fine with me 🤷🏻